pkg/app/app.go

```diff
@@ -37,11 +37,11 @@ type App struct {
 	Updater   *updates.Updater // may only need this on the Gui
 }
 
-func Run(config config.AppConfigurer, common *common.Common, filterPath string) {
+func Run(config config.AppConfigurer, common *common.Common, filterPath string, gitArgs string) {
```

Comment:
given that we're passing both filterPath and gitArgs around quite a bit from main to our gui struct, could we wrap them in a struct called MainArgs ? This ensures that if we want to pass more info from main to our gui struct we won't need to touch as many parts of the code.

Reply:
That makes sense :thumbsup: I'll try.